### PR TITLE
Oxford colorectal handler

### PR DESCRIPTION
**Pull request by FrancescoNea**
Colorectal Handler for Oxford Provider

### CODE REVIEW

File: test/lib/import/colorectal/providers/royal_marsden/royal_marsden_handler_colorectal_test.rb [hunk 1]
```diff
@@ -55,7 +55,7 @@ def setup
 
   test 'process_test_scope' do
     @handler.process_test_scope(@genotype, @record)
-    assert_equal 'Targeted Colorectal mutation test', @genotype.attribute_map['genetictestscope']
+    assert_equal 'Targeted Colorectal Lynch or MMR', @genotype.attribute_map['genetictestscope']
```
Comment (ollietulloch) [inline test/lib/import/colorectal/providers/royal_marsden/royal_marsden_handler_colorectal_test.rb:58]: Hi @FrancescoNea - this doesn't look related to the Oxford colorectal handler?

Comment (FrancescoNea) [reply test/lib/import/colorectal/providers/royal_marsden/royal_marsden_handler_colorectal_test.rb:58]: It is part of a broader change of labels I had to perform. In Oxford, Ashkenazi Jews and Polish ethnicities are tested as well, so had to re-label AJ and Polish screens accordingly to the Colorectal tests, in order to differentiate them from the BRCA1/2 tests

Comment (ollietulloch) [reply test/lib/import/colorectal/providers/royal_marsden/royal_marsden_handler_colorectal_test.rb:58]: Thanks @FrancescoNea

File: test/lib/import/colorectal/providers/oxford/oxford_handler_colorectal_test.rb [hunk 2]
```diff
@@ -0,0 +1,130 @@
+require 'test_helper'
+
+class OxfordHandlerTest < ActiveSupport::TestCase
+  def setup
+    @record   = build_raw_record('pseudo_id1' => 'bob')
+    @genotype = Import::Brca::Core::GenotypeBrca.new(@record)
+    # TODO: Fully qualify CambridgeHandler in cambridge_handler.rb
```
Comment (ollietulloch) [inline test/lib/import/colorectal/providers/oxford/oxford_handler_colorectal_test.rb:11]: Hi @FrancescoNea, this comment doesn't look related to the Oxford handler?

Comment (FrancescoNea) [reply test/lib/import/colorectal/providers/oxford/oxford_handler_colorectal_test.rb:11]: Hi @ollietulloch ,

That's a "typo", apologies.

Oxford Colorectal Coverage test was created from a template for Oxford BRCA coverage test, which in turn was created from Cambridge BRCA coverage test.

Comment (ollietulloch) [reply test/lib/import/colorectal/providers/oxford/oxford_handler_colorectal_test.rb:11]: Thanks @FrancescoNea - no problem, could you push a corrective commit to this branch please?

Comment (FrancescoNea) [reply test/lib/import/colorectal/providers/oxford/oxford_handler_colorectal_test.rb]: Hi @ollietulloch ,
Resolved by commit [846a3fa](https://github.com/publichealthengland/data_management_system/pull/94/commits/846a3fa7e99e59d877549ccc64a9306c518a83a4)

Comment (ollietulloch) [reply test/lib/import/colorectal/providers/oxford/oxford_handler_colorectal_test.rb]: Great, thanks @FrancescoNea

**Review by ollietulloch — CHANGES_REQUESTED** [review]
Hi @FrancescoNea, 

could you please add test coverage for the `assign_genomic_change` method.

Thanks

**Review by ollietulloch — APPROVED** [review]

**Review by ollietulloch — APPROVED** [review]
Thanks @FrancescoNea 

Something to consider later is perhaps removing the logging behaviour, or ensure it's only enabled in developement, so that we can remove the `capture_io` behaviour in tests.
